manim/constants.py

```diff
@@ -2,140 +2,141 @@
 Constant definitions.
 """
 
-import numpy as np
+import typing
 
+import numpy as np
 
 # Messages
-NOT_SETTING_FONT_MSG = """
+NOT_SETTING_FONT_MSG: str = """
 You haven't set font.
 If you are not using English, this may cause text rendering problem.
 You set font like:
 text = Text('your text', font='your font')
 """
-SCENE_NOT_FOUND_MESSAGE = """
+SCENE_NOT_FOUND_MESSAGE: str = """
    {} is not in the script
 """
-CHOOSE_NUMBER_MESSAGE = """
+CHOOSE_NUMBER_MESSAGE: str = """
 Choose number corresponding to desired scene/arguments.
 (Use comma separated list for multiple entries)
 Choice(s): """
-INVALID_NUMBER_MESSAGE = "Invalid scene numbers have been specified. Aborting."
-NO_SCENE_MESSAGE = """
+INVALID_NUMBER_MESSAGE: str = "Invalid scene numbers have been specified. Aborting."
+NO_SCENE_MESSAGE: str = """
    There are no scenes inside that module
 """
 
 # Cairo and Pango stuff
-NORMAL = "NORMAL"
-ITALIC = "ITALIC"
-OBLIQUE = "OBLIQUE"
-BOLD = "BOLD"
+NORMAL: str = "NORMAL"
+ITALIC: str = "ITALIC"
+OBLIQUE: str = "OBLIQUE"
+BOLD: str = "BOLD"
 # Only for Pango from below
-THIN = "THIN"
-ULTRALIGHT = "ULTRALIGHT"
-LIGHT = "LIGHT"
-SEMILIGHT = "SEMILIGHT"
-BOOK = "BOOK"
-MEDIUM = "MEDIUM"
-SEMIBOLD = "SEMIBOLD"
-ULTRABOLD = "ULTRABOLD"
-HEAVY = "HEAVY"
-ULTRAHEAVY = "ULTRAHEAVY"
+THIN: str = "THIN"
+ULTRALIGHT: str = "ULTRALIGHT"
+LIGHT: str = "LIGHT"
+SEMILIGHT: str = "SEMILIGHT"
+BOOK: str = "BOOK"
+MEDIUM: str = "MEDIUM"
+SEMIBOLD: str = "SEMIBOLD"
+ULTRABOLD: str = "ULTRABOLD"
+HEAVY: str = "HEAVY"
+ULTRAHEAVY: str = "ULTRAHEAVY"
 
 
 # Geometry: directions
-ORIGIN = np.array((0.0, 0.0, 0.0))
+ORIGIN: np.ndarray = np.array((0.0, 0.0, 0.0))
 """The center of the coordinate system."""
 
-UP = np.array((0.0, 1.0, 0.0))
+UP: np.ndarray = np.array((0.0, 1.0, 0.0))
 """One unit step in the positive Y direction."""
 
-DOWN = np.array((0.0, -1.0, 0.0))
+DOWN: np.ndarray = np.array((0.0, -1.0, 0.0))
 """One unit step in the negative Y direction."""
 
-RIGHT = np.array((1.0, 0.0, 0.0))
+RIGHT: np.ndarray = np.array((1.0, 0.0, 0.0))
 """One unit step in the positive X direction."""
 
-LEFT = np.array((-1.0, 0.0, 0.0))
+LEFT: np.ndarray = np.array((-1.0, 0.0, 0.0))
 """One unit step in the negative X direction."""
 
-IN = np.array((0.0, 0.0, -1.0))
+IN: np.ndarray = np.array((0.0, 0.0, -1.0))
 """One unit step in the negative Z direction."""
 
-OUT = np.array((0.0, 0.0, 1.0))
+OUT: np.ndarray = np.array((0.0, 0.0, 1.0))
 """One unit step in the positive Z direction."""
 
 # Geometry: axes
-X_AXIS = np.array((1.0, 0.0, 0.0))
-Y_AXIS = np.array((0.0, 1.0, 0.0))
-Z_AXIS = np.array((0.0, 0.0, 1.0))
+X_AXIS: np.ndarray = np.array((1.0, 0.0, 0.0))
```

Comment:
Same here, I don't think this needs a type hint.
And actually, all the directions function very well with normal lists (no np.arrays) so the typehints could be misleading